app/views/activity_insight_oa_workflow/permissions_curation/index.html.erb

```diff
@@ -1,7 +1,6 @@
 <div class="container pt-4 pb-5">
   <%= link_to '<< Back', activity_insight_oa_workflow_path %>
   <h1 class="pb-3 pt-2">Publications Requiring Permissions Review</h1>
-  <div class='card'>
   <table class="table table-scrollable">
```

Comment:
If we remove the card container, we should remove table-scrollable class.  It will probably be more useful for admins to be able to see the whole table without scrolling.

Reply:
Yeah, my thought was that if the tables are getting wide, then better to allow them to grow to the size of the screen rather than hiding part of them inside of a container with a fixed size. If they get so wide that you have to scroll the whole screen, then maybe that's another issue, but we're not there yet...